assemblies/assembly_openshift-managed/main.adoc

```diff
@@ -16,10 +16,14 @@ to provide an infrastructure to execute processes and other business assets. You
 
 .Prerequisites
 
-* At least four gigabytes of memory are available in the OpenShift environment.
+* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
 * The OpenShift project for the deployment is created.
 * You have logged in to the project using the OpenShift web console and using the `oc` command.
-* If you intend to scale any of the {CENTRAL} or {CENTRAL} Monitoring pods, your OpenShift environment supports persistent volumes with ReadWriteMany mode.
+* Dynamic persistent volume (PV) provisioning is enabled. Alternatively, enough persistent volumes are available. By default, the following sizes are required:
+** Each deployed replicated set of {KIE_SERVER} pods requires one 1Gi PV for the database. This requirement does not apply if you use an external database server.
```

Comment:
it is not a requirement, is a customizable field.
this requirement is only for OpenShift Online and was decided to use this value.

Reply:
Changed throughout.